tests/test_extensions.py

```diff
@@ -0,0 +1,56 @@
+# This file is part of astro_metadata_translator.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the LICENSE file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# Use of this source code is governed by a 3-clause BSD-style
+# license that can be found in the LICENSE file.
+
+import pickle
+import unittest
+from astro_metadata_translator import ObservationInfo, makeObservationInfo, PropertyDefinition
+
+"""Test that extensions to the core set of properties works"""
```

Comment:
Can we have a test where you:

* Create a little translator class that defines and extension and an extension translation method (maybe a trivial_map and an explicit to_ method)
* Translate a simple header
* Check that  the extenion properties are filled in.

I think this could be done with a very small change to the test_translation.py test file (that uses `InstrumentTestTranslator`).